libr/anal/jmptbl.c

```diff
@@ -33,11 +33,11 @@ static void queue_case(RAnal *anal, ut64 switch_addr, int offset_sz, ut64 case_a
 // analyze a jmptablle inside a function // maybe rename to r_anal_fcn_jmptbl() ?
 R_API bool r_anal_jmptbl(RAnal *anal, RAnalFunction *fcn, ut64 jmpaddr, ut64 table, ut64 tablesize, ut64 default_addr) {
 	const int depth = 50;
-	return try_walkthrough_jmptbl (anal, fcn, depth, jmpaddr, table, table, tablesize, tablesize, default_addr, 0);
+	return try_walkthrough_jmptbl (anal, fcn, depth, jmpaddr, table, table, tablesize, tablesize, default_addr, false);
 }
 
-R_API int try_walkthrough_jmptbl(RAnal *anal, RAnalFunction *fcn, int depth, ut64 ip, ut64 jmptbl_loc, ut64 jmptbl_off, ut64 sz, int jmptbl_size, ut64 default_case, int ret0) {
-	int ret = ret0;
+R_API bool try_walkthrough_jmptbl(RAnal *anal, RAnalFunction *fcn, int depth, ut64 ip, ut64 jmptbl_loc, ut64 jmptbl_off, ut64 sz, int jmptbl_size, ut64 default_case, bool ret0) {
```

Comment:
are you sure this change is ok? I see `try_walkthrough_jmptbl` being called with `ret0=4` in https://github.com/radareorg/radare2/blob/master/libr/anal/fcn.c#L841

Reply:
in fact this argument is kind o f  stupid because its just returned directly from the  funnction and i ts not used anywhere so we can remove ths paramenter i think, but not in this pr

Reply:
yeah